frontend/src/App.js

```diff
@@ -99,14 +103,36 @@ const App = () => {
 
         addMessage("Manager: Played phrase!");
     };
+
+    const customerBaseImages = [
+        "/images/customers/customer1.png",
+        "/images/customers/customer2.png",
+        "/images/customers/customer3.png",
+        "/images/customers/customer4.png",
+        "/images/customers/customer5.png"
+    ];
     const handleReceiveOrder = () => {
-        setOrderVisible(true);
+        setOrderVisible(false);
+        setBurger(null);
+        setSide(null);
+        setDrink(null);
         getBurgerOrder();
         getDrinkOrder();
         getSideOrder();
-        const randomIndex = Math.floor(Math.random() * customerImages.length);
-        const randomCustomer = customerImages[randomIndex];
-        setCustomerImage(randomCustomer);
+
+        const randIndex = Math.floor(Math.random() * customerBaseImages.length);
+        const randomCustomer = customerBaseImages[randIndex];
+        setBaseCustomerImage(randomCustomer);
+        setCurrentCustomerImage(randomCustomer);
+
+        console.log("Random customer selected:", randomCustomer);
+
+        setTimeout(() => {
+            setOrderVisible(true);
+            addMessage("Manager: Receiving the order...");
+        }, 3000);
```

Comment:
making this randomly generated would add some variability to make it feel less robotic